public/static/locales/eng/terms.md

```diff
@@ -1,51 +1,49 @@
-# Terms and conditions
+# Terms and Conditions
 
-NOTE: Translation efforts are in progress. Until then, please direct any questions to samromur@ru.is.
+On this website you can take part in the creation of the Samrómur database. By participating, you submit your own voice recording and, as appropriate, demographic information, reviews, and feedback that will be useful in creating an open voice database.
 
-Á vefsvæðinu [samrómur.is](https://www.samromur.is) getur þú tekið þátt í gerð gagnasafnsins Samrómur. Í þátttöku þinni felst að þú skilar inn eigin raddupptöku og eftir atvikum, lýðfræðilegum upplýsingum, yfirferðum og endurgjöf sem mun nýtast við gerð opins radd gagnasafn.
+The database will be accessible at [www.samromur.is](www.samromur.is/gagnasafn). With the creation of the voice database, it will be possible to support and strengthen the development of language technology solutions for Icelandic.
```

Comment:
```suggestion
The database will be accessible at [www.samromur.is](https://samromur.is/gagnasafn). With the creation of the voice database, it will be possible to support and strengthen the development of language technology solutions for Icelandic.
```
It shows up as a relative link unless there is https